src/components/Shared/SendModal/SendOnChain/SendOnChain.test.tsx

```diff
@@ -70,20 +70,16 @@ describe("SendOnChain", () => {
       "wallet.address"
     ) as HTMLInputElement;
 
-    await act(async () => {
-      userEvent.clear(addressInput);
-      userEvent.type(addressInput, "abc123456789");
-    });
+    userEvent.clear(addressInput);
```

Comment:
userEvent stuff doesn't need to be wrapped in `act`. 
Kind of a weird error message. Look at the other changes and you see what the actual problem was => using `waitFor` & `findByText` instead of `getByText` to await the state changes fixes the tests. :)